docs/html/user_guide.rst

```diff
@@ -772,6 +772,172 @@ archives are built with identical packages.
 
 .. _`Using pip from your program`:
 
+Fixing conflicting dependencies
+===============================
+
+The purpose of this setion of documentation is to provide practical suggestions to
+pip users who encounter an error where pip cannot install their
+specified packages due to conflicting dependencies (a
+``ResolutionImpossible`` error).
+
+This documentation is specific to the new resolver, which you can use
+with the flag ``--unstable-feature=resolver``.
+
+Understanding your error message
+--------------------------------
+
+When you get a ``ResolutionImpossible`` error, you might see something
+like this:
+
+::
+
+    pip install packageA==0.44.1 packageB==4.3.0
```

Comment:
I wonder if this is to be compatible with other documents, otherwise as an user I'd be really happy to *not* having to see camelCase distribution names.  I also wonder if we want to keep using *packages* to refer to *distributions* (a distribution may contain multiple packages), although I find both equally unclear and have no better alternative to offer.

Reply:
For clarity and readability, to help the user easily distinguish between the two example packages, what could maybe jump out more? Maybe something like `package_carrot` and `package_zebra`?

I'm fine with referring to distributions as packages
1. in a user guide *when the distinction doesn't super-matter* (communicate in the user's language when possible)
2. for right now, in the interests of expedience. We can polish the exact word choice iteratively.

Overall: I'm very willing to sacrifice pedantic consistency in the interest of writing accessibly.

Reply:
Going to use "package_coffee" and "package_tea" that both depend on "package_water".